modules/next/modules/next_graphql/next_graphql.info.yml

```diff
@@ -1,7 +1,7 @@
 name: 'Next.js GraphQL'
 type: module
 description: 'GraphQL for Next.js'
-core_version_requirement: ^9 || ^10
+core_version_requirement: ^10.1 || ^11
```

Comment:
@apathak18 Can we keep `core_version_requirement` uniform across all module/sub-modules. Ex: Here we've added `^10.1` and in `next_extras` sub-module we have' added `^10`.